.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Please see CONTRIBUTING.md for guidelines. -->
+
+## Environment
+
+* SuperCollider version:
+* Operating system:
+
+## Steps to reproduce
+
+<!-- Use triple backticks ``` ``` to delimit code. Make sure to use a minimal example. -->
```

Comment:
we can see how it plays out but I expect some people will misinterpret this and write triple backticks around inline code. the previous version of this was nice in that it demonstrated both the expected form and that you could use triple-backtick followed by 'supercollider' to get the syntax highlighting.